Code/Editor/AzAssetBrowser/AzAssetBrowserWindow.cpp

```diff
@@ -113,6 +113,7 @@ AzAssetBrowserWindow::AzAssetBrowserWindow(QWidget* parent)
     AZ_Assert(m_assetBrowserModel, "Failed to get filebrowser model");
     m_filterModel->setSourceModel(m_assetBrowserModel);
     m_filterModel->SetFilter(m_ui->m_searchWidget->GetFilter());
+    m_filterModel->setDynamicSortFilter(false);
```

Comment:
I think false is the default. Also, could you comment as to why this is disabled? Something like "we'll call sort ourselves" or whatever...

Reply:
Done